mapreduce/tests/test_integration.py

```diff
@@ -12,7 +12,20 @@
 def test_check(aggregator, check, instance):
```

Comment:
I'd suggest renaming this file to test_check and this test method to test_integration. Also add @pytest.mark.integration to it

Reply:
there already exists a `test_mapreduce.py`, should I rename to `test_integration_e2e.py`?

Reply:
ok